js/pihole/index.js

```diff
@@ -154,13 +166,14 @@ function updateSummaryData(runOnce) {
 }
 
 function updateQueriesOverTime() {
-    $.getJSON("api.php?overTimeData", function(data) {
+    $.getJSON("api.php?overTimeData10mins", function(data) {
         // Add data for each hour that is available
         // remove last data point since it not representative
         data.ads_over_time.splice(-1,1);
-        for (hour in data.ads_over_time) {
-            // Add x-axis label
-            timeLineChart.data.labels.push(hour + ":00");
+        for (var hour in data.ads_over_time) {
+            var d = new Date();
+            d.setHours(Math.floor(hour/6),10*(hour%6),0,0);
+            timeLineChart.data.labels.push(d);
```

Comment:
Okay, look at this update. I just always takes some time to identify what specific setting would be necessary with Chart.js (needless to mention that many links in their docs actually point to nowhere).